ledger-tool/src/main.rs

```diff
@@ -381,7 +383,26 @@ fn output_account(
     println!("  rent_epoch: {}", account.rent_epoch());
     println!("  data_len: {}", account.data().len());
     if print_account_data {
-        println!("  data: '{}'", bs58::encode(account.data()).into_string());
+        if encoding == UiAccountEncoding::Base58 {
+            println!("  data: '{}'", bs58::encode(account.data()).into_string());
+            println!("  encoding: base58");
+        } else {
+            let account_data = UiAccount::encode(pubkey, account, encoding, None, None).data;
+            match account_data {
+                UiAccountData::Binary(data, data_encoding_str) => {
+                    println!("  data: '{}'", data);
+                    println!("  encoding: {:?}", data_encoding_str);
```

Comment:
One last nit: the derived debug implementation of `UiAccountEncoding` will not be in camel case, so this will be inconsistent. There are different ways to do it, but since we're using serde_json a lot, probably easiest to just:
```suggestion
                    println!("  encoding: {}", serde_json::from_str(data_encoding_str).unwrap());
```

Reply:
I dont think that would work from_str expects a &str type.

Reply:
Sorry, bad copy: `serde_json::to_string`

Reply:
to_string would work. I can make that change

Reply:
would camlCase matter here . as only thing that is being printed is `base64` and `base64+zstd`

Reply:
Oh I see what you mean we will get `Base64` and `Base64Zstd` instead